tldw_Server_API/app/api/v1/endpoints/vector_stores_openai.py

```
@@ -1554,7 +1553,7 @@ class CreateFromMediaRequest(BaseModel):
 async def create_store_from_media(
     payload: CreateFromMediaRequest = Body(...),
     current_user: User = Depends(get_request_user),
-    db: MediaDatabase = Depends(get_media_db_for_user)
+    db: Any = Depends(get_media_db_for_user)
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `db`

(ANN401)